src/modal/Modal.tsx

```diff
@@ -183,7 +199,14 @@ export class Modal extends Component<ModalProps> {
                       <CrossIcon size="18" iconColor={color.blue} />
                     </Button>
                   )}
-                  <div className={`${baseClassName}-body`}>{this.props.children}</div>
+                  <div
+                    className={cc([
+                      `${baseClassName}-body`,
+                      this.state.showFooterBorder ? `${baseClassName}-modalFooterBorder` : '',
```

Comment:
I'm not sure to understand where this class (`${baseClassName}-modalFooterBorder`) is defined.

Reply:
Explained here: https://github.com/blablacar/ui-library/pull/817#discussion_r460950021